.circleci/config.yml

```diff
@@ -5,46 +5,38 @@ shared_configs:
         name: Run tests
         command: |
           make deps test
-
+  full_job_steps: &full_job_steps
+    - checkout
+    - run:
+        name: Run tests
+        command: |
+          make ci
 
 # Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
 version: 2.1
 jobs:
   build-1-15:
     working_directory: ~/repo
     docker:
-      - image: circleci/golang:1.15
+      - image: cimg/go:1.15
     steps: *simple_job_steps
 
   build-1-16:
     working_directory: ~/repo
     docker:
-      - image: circleci/golang:1.16
-    steps:
-      - checkout
-      - restore_cache:
-          keys:
-            - go-mod-v4-{{ checksum "go.sum" }}
-      - run:
-          name: Install Dependencies
-          command: go mod download
-      - save_cache:
-          key: go-mod-v4-{{ checksum "go.sum" }}
-          paths:
-            - "/go/pkg/mod"
-      - run:
-          name: Run tests
-          command: |
-            #mkdir -p /tmp/test-reports
-            #gotestsum --junitfile /tmp/test-reports/unit-tests.xml
-            make ci
-    #- store_test_results:
-    #    path: /tmp/test-reports
+      - image: cimg/go:1.16
+    steps: *full_job_steps
```

Comment:
do you know why only build-1-16 is using full_job_steps?

Reply:
I don't; I get the idea of only running the full deal on 1/4 steps, but I'm not sure why Go 1.16 was specifically chosen for this.